frontend/src/components/enroll/enroll.css

```diff
@@ -0,0 +1,77 @@
+* {
```

Comment:
Let's move this to the `static` directory and have it as a shared CSS file between all the static pages.